docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_the_cli.rst

```diff
@@ -73,7 +79,7 @@ How to create a new Expectation Suite using the CLI
 
         .. code-block:: bash
 
-            great_expectations --v3-api suite new --no-jupyter
+            great_expectations --v3-api suite new --non-interactive --no-jupyter
```

Comment:
@anthonyburdi This is a little strange -- should we not be able to do `--no-jupyter` without saying `--non-interactive`, too?

Reply:
Yep! The examples here are for the "being explicit" cases.